tools/get_coverage_ros2_pkg.py

```diff
@@ -0,0 +1,116 @@
+#!/usr/bin/env python3
+
+import argparse
+import os
+import requests
+import subprocess
+import sys
+import tempfile
+
+
+# Compare if a list is inside other list in order
+# From here: https://stackoverflow.com/a/20789669
+def is_slice_in_list(s, l):
+    len_s = len(s)
+    return any(s == l[i:len_s + i] for i in range(len(l) - len_s + 1))
+
+
+def get_src_path(package_name, colcon_ws):
+    cmd = ['colcon', 'list', '--paths-only', '--base-paths', colcon_ws, '--packages-select', package_name]
+    try:
+        path = subprocess.check_output(cmd).decode('ascii').strip()
+    except subprocess.CalledProcessError as e:
+        print(e.output, file=sys.stderr)
+        sys.exit(-1)
+    if not path:
+        print('Package not found: ' + input_pkg, file=sys.stderr)
+        sys.exit(-1)
+    assert path.count('\n') == 0, 'Found more than one line returned by colcon list'
```

Comment:
I would suggest to use `len(path.splitlines())` to be platform / newline agnostic and not care about trailing newlines.

Reply:
better, thanks 95415ba